assets/_sass/_structure.scss

```diff
@@ -80,13 +80,23 @@
         position: absolute;
         padding: 0px 20px;
         top: 12px;
-        left: -48px;
+        left: -70px;
+        height: 100%;
+        
+        @media (max-width: $grid-float-breakpoint) {
+          left: unset;
+          right: 0px;
+        }
       }
 
       .article-title {
         display: flex;
         align-items: center;
 
+        @media (max-width: $grid-float-breakpoint) {
+          max-width: calc(100vw - 120px);
```

Comment:
nit++: it feels wrong to use pixels and viewport fractions together, even if you won't encounter a 120px wide display

Reply:
@effleurager the reason for this is I need to limit the width of the article headers to give enough space for the permalink and article option buttons. Hence 100% of the viewport width minus 120px. The 120px accounts for all of these elements.